src/contracts/minter/MinterTypes.sol

```diff
@@ -83,19 +83,21 @@ struct MinterParams {
 }
 
 /**
- * @notice Information on a token that is a Kresko asset.
- * @dev Each Kresko asset has 18 decimals.
- * @param kFactor The k-factor used for calculating the required collateral value for Kresko asset debt.
- * @param oracle The oracle that provides the USD price of one Kresko asset.
- * @param supplyLimit The total supply limit of the Kresko asset.
+ * @notice Information on a token that is a KreskoAsset.
+ * @dev Each KreskoAsset has 18 decimals.
+ * @param kFactor The k-factor used for calculating the required collateral value for KreskoAsset debt.
+ * @param oracle The oracle that provides the USD price of one KreskoAsset.
+ * @param supplyLimit The total supply limit of the KreskoAsset.
  * @param closeFee The percentage paid in fees when closing a debt position of this type.
- * @param exists Whether the Kresko asset exists within the protocol.
- * @param mintable Whether the Kresko asset can be minted.
+ * @param anchor The anchor address
+ * @param exists Whether the KreskoAsset exists within the protocol.
+ * @param mintable Whether the KreskoAsset can be minted.
```

Comment:
Will need to remove KrAsset's `mintable` attribute for compatibility with latest `main` branch